test-apps/tutorial-servo-can-control/src/main.cpp

```diff
@@ -0,0 +1,29 @@
+#include "CANMsg.h"
+#include "TutorialServo.h"
+#include "mbed.h"
+
+// servo PWM control pin is connected to PA_1
+PwmOut servoPwmOut(PA_1);
+
+// assume RX pin is PB_8 and TX pin is PB_9
+CAN can(PB_8, PB_9);
+
+// note that txMsg is not declared since I'm not sending packets
+CANMsg rxMsg;
+
+int main() {
+  // pass no paramter to TutorialServo's constructor, using 1ms ~ 2ms and 0 ~ 180 degrees as the range mapping
+  TutorialServo servo = TutorialServo();
```

Comment:
Declare the Servo library outside the `main` function. Your object declaration doesn't align with the right syntax. Check this [tut](https://www.w3schools.com/cpp/cpp_constructors.asp) for more examples on how to pass parameters to the constructor. Also you will have to pass the servopin to the constructor.